js/main.js

```diff
@@ -0,0 +1,49 @@
+let user = "John Doe";
+let student;
+
+console.log(user); //"John Doe"
+console.log(student); //"Undefined"
+
+let student = "Oleg";
```

Comment:
`student` was defined on line 2, remove `let` operator.

Reply:
I fixed it.